openapi/anchor-platform/bundled_callback.yml

```diff
@@ -711,6 +731,335 @@ components:
+          enum:
+            - receive
+        status:
+          $ref: '#/components/schemas/StatusSEP31'
+        amount_expected:
+          $ref: '#/components/schemas/Amount'
+        amount_in:
+          $ref: '#/components/schemas/Amount'
+        amount_out:
+          $ref: '#/components/schemas/Amount'
+        amount_fee:
+          $ref: '#/components/schemas/Amount'
+        quote_id:
+          type: string
+        started_at:
+          type: string
+          format: date-time
+        updated_at:
+          type: string
+          format: date-time
+        completed_at:
+          type: string
+          format: date-time
+        transfer_received_at:
+          type: string
+          format: date-time
+        message:
+          type: string
+        refunds:
+          $ref: '#/components/schemas/Refunds'
+        stellar_transactions:
+          type: array
+          items:
+            $ref: '#/components/schemas/StellarTransaction'
+        external_transaction_id:
+          type: string
+        customers:
+          type: object
+          description: |
+            The Identification info of the sending and receiving customers. If they were created through [SEP-12](https://stellar.org/protocol/sep-12),
+            this object should contain the SEP-12 customer `id`. Otherwise, the `account` address of the customer.
+          properties:
+            sender:
+              $ref: '#/components/schemas/StellarId'
+            receiver:
+              $ref: '#/components/schemas/StellarId'
+        creator:
+          $ref: '#/components/schemas/StellarId'
+    Quote:
+      type: object
+      properties:
+        id:
+          type: string
+        sell_amount:
+          type: string
+        sell_asset:
+          type: string
+        buy_amount:
+          type: string
+        buy_asset:
+          type: string
+        expires_at:
+          type: string
+          format: date-time
+        price:
+          type: string
+        total_price:
+          type: string
+        creator:
+          $ref: '#/components/schemas/StellarId'
+        transaction_id:
+          type: string
+        created_at:
+          type: string
+          format: date-time
+    Event:
+      type: object
+      properties:
+        id:
+          type: string
+        type:
+          type: string
+          enum:
+            - transaction_created
+            - transaction_status_changed
+            - transaction_error
+            - quote_created
+          description: |
+            The transaction event type. Can be one of the following:
+            - `transaction_created` - a transaction was created through the SEP endpoints. The payload is in the `transaction` field.
+            - `transaction_status_changed` - the status of a transaction has changed. The payload is in the `transaction` field.
+            - `transaction_error` - error processing the transaction. The payload is in the `transaction` field.
```

Comment:
If it's an error, how do we retrieve an error itself?

Reply:
`transaction_error` means one of two things

1. the anchor platform experienced an unexpected error and put the transaction in the `error` status as result. If this happens, the business has to triage the issue and if possible, move it back out of the error status and potentially updating other fields so that it can continue processing. At best, the platform will have put an informative message in the `message` field as a starting point.
2. the business put the transaction into the error status itself via `PATCH /transactions` or the associated RPC method. In this case, the business knows what the error was.

I have a 3.0 jira issue for merging `transaction_error` event types into the `transaction_status_changed` type.

Reply:
I think it's a good idea to put this message in the OpenAPI file spec

Reply:
I think it is, I see the message field on in the docs.